packages/webapp/src/containers/Animals/Inventory/KPI/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import clsx from 'clsx';
+import { useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation, TFunction } from 'react-i18next';
+import {
+  useGetDefaultAnimalTypesQuery,
+  useGetCustomAnimalTypesQuery,
+} from '../../../../store/api/apiSlice';
+import { PureTileDashboard, TypeCountTile } from '../../../../components/TileDashboard';
+import useQueries from '../../../../hooks/api/useQueries';
+import { CustomAnimalType, DefaultAnimalType } from '../../../../store/api/types';
+import { ReactComponent as CattleIcon } from '../../../../assets/images/animals/cattle-icon.svg';
+import { ReactComponent as ChickenIcon } from '../../../../assets/images/animals/chicken-icon.svg';
+import { ReactComponent as PigIcon } from '../../../../assets/images/animals/pig-icon.svg';
+import { ReactComponent as CustomIcon } from '../../../../assets/images/nav/animals.svg';
+import { uppercaseTheFirstLetter } from '../../../../util';
+import { getComparator } from '../../../../util/sort';
+import { generateUniqueAnimalTypeId } from '../../../../util/animal';
+import styles from './styles.module.scss';
+
+const icons = {
+  CATTLE: <CattleIcon />,
+  CHICKEN_BROILERS: <ChickenIcon />,
+  CHICKEN_LAYERS: <ChickenIcon />,
+  PIGS: <PigIcon />,
+  '': <CustomIcon />,
+};
+
+const formatAnimalTypes = (
+  types: (DefaultAnimalType | CustomAnimalType)[],
+  onTypeClick: (typeId: string) => void,
+  t: TFunction,
+) => {
+  const formattedTypes: TypeCountTile[] = [];
+  types.forEach((type) => {
+    if (type.count) {
+      const label = 'type' in type ? type.type : t(`animalTypes:${type.key}`);
+      const iconKey = ('key' in type ? type.key : '') as keyof typeof icons;
+      const id = generateUniqueAnimalTypeId(type);
+
+      formattedTypes.push({
+        label: uppercaseTheFirstLetter(label),
```

Comment:
It's a good utility function that we should for sure keep available, but I wonder if `text-transform: capitalize` might look better on tiles?

<img width="230" alt="Screenshot 2024-02-29 at 5 13 47 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/1f1a7500-8a1d-4bf5-8e27-b687a1b214dc">


Did Loïc express a preference? I think Figma was all one-word types so it's hard to judge from there.

Reply:
Ohhh! I've gotten so used to `toLocaleCompare()` I totally forgot that part of native JS comparison on strings! Got it 👍